docs/pages/versions/v46.0.0/sdk/sensors.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Sensors
-sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-46/packages/expo-sensors'
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-sensors'
```

Comment:
This needs to stay this way - the `sdk-46` branch will be created during SDK release 😉 
```suggestion
sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-46/packages/expo-sensors'
```